src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/decisionissued/DecisionIssuedMidEventHandlerTest.java

```diff
@@ -176,8 +176,9 @@ public void givenCaseWithAppointee_thenCorrectlySetTheNoticeNameWithAppellantAnd
 
         handler.handle(MID_EVENT, callback, USER_AUTHORISATION);
 
-        verifyTemplateBody(NoticeIssuedTemplateBody.ENGLISH_IMAGE, "Appointee Surname, appointee for Appellant Lastname",
-                documentConfiguration.getDocuments().get(LanguagePreference.ENGLISH).get(EventType.DECISION_ISSUED));
+        verifyTemplateBody(NoticeIssuedTemplateBody.ENGLISH_IMAGE,
+                "Appointee Surname, appointee for Appellant Lastname",
```

Comment:
Since you've gone to the trouble, it would be best to put every arg on a new line